src/StudyEntities/Candidate/Sex.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+/**
+ * This defines valid values for the Sex data point for a Candidate.
+ * It is a Value Object used to assist with validation.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  LORIS
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\StudyEntities\Candidate;
+
+/**
+ * This defines valid values for the Sex data point for a Candidate.
+ * It is a Value Object used to assist with validation.
+ *
+ * @category Main
+ * @package  LORIS
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Sex
+{
+    /* @var string */
+    public $value;
+
+    private const VALID_VALUES = array(
+                                  'male',
+                                  'female',
+                                  'other',
+                                 );
+
+    /**
+     * Calls validate() immediately.
+     *
+     * @param string $value The value of this EntityType.
+     *
+     * @throws \DomainException When the value is not valid
+     */
+    final public function __construct(string $value)
```

Comment:
There's no reason for this to be final.

Reply:
The parent class (written by @xlecours) defines it that way.

Reply:
This also doesn't have a parent class, and if it *did* have a parent class with a final constructor that would make this code illegal because `final` means "can't be overloaded".

Reply:
You're right I was mistaken